docs/Concepts/Consensus-Protocols/Overview-Consensus.md

```diff
@@ -1,50 +1,48 @@
 description: Besu consensus protocols
 <!--- END of page meta data -->
 
-# Consensus Protocols 
+# Consensus protocols
 
-Besu implements a number of consensus protocols: 
+Besu implements a number of consensus protocols, including:
```

Comment:
I'm not sure if this is just me but does adding _including_ here make it sound like the consensus protocols listed here are a subset of consensus protocols implemented in Besu rather than being all of them?

Reply:
Updated.